README.md

```diff
@@ -344,14 +324,14 @@ to run phing tests and metrics.
 Debugging, logging
 ==================
 
-You need log4php (http://logging.apache.org/log4php/). Just make sure you copy
-it to the include_path and PAMI will pick it up from there.
+You will need a PSR3 compatible logger. By default, PAMI will use log4php
+(http://logging.apache.org/log4php/).
```

Comment:
```markdown
...will use [log4php](http://logging.apache.org/log4php/), for which an adapter is provided.
```

Just to make sure that people won't get confused :smile:. Just a suggestion though.